src/apps/intel/intel10g.lua

```diff
@@ -19,13 +19,16 @@ local index_set = require("lib.index_set")
 local macaddress = require("lib.macaddress")
 local mib = require("lib.ipc.shmem.mib")
 local timer = require("core.timer")
+local counter = require("core.counter")
 
 local bits, bitset = lib.bits, lib.bitset
 local band, bor, lshift = bit.band, bit.bor, bit.lshift
 
 num_descriptors = 512
 --num_descriptors = 32
 
+local ifInDiscards = counter.open("nic/ifInDiscards")
```

Comment:
Probably not the right thing for us, given that we need to instantiate multiple NICs.  Marcel can get away with just this one because he runs the lwAFTR on a stick.